### PR TITLE
tools nerf

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
reduces tool health by 20%
Tools now all are able to break unless you have nanorepair
Makes auro damper now increase degeration rather then decresses tool break tool threashold

### CODE REVIEW

**Comment by Dissentbay** [comment]
makes melee weapons feel like they are made of glass
while I like the idea of making overmodding even worse, that amount might be too much

**Comment by Trilbyspaceclone** [comment]
> makes melee weapons feel like they are made of glass while I like the idea of making overmodding even worse, that amount might be too much

this hasnt even been test merged

File: code/game/objects/items/weapons/tools/_tools.dm [hunk 1]
```diff
@@ -47,8 +47,8 @@
 	//Variables used for tool degradation
 	var/degradation = 0.8 //If nonzero, the health of the tool decreases by this amount after each tool operation
 	health = 0		// Health of a tool.
-	max_health = 500
-	var/health_threshold  = 40 // threshold in percent on which tool health stops dropping
+	max_health = 250
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/_tools.dm:50]:
```suggestion
	max_health = 400
```

**Comment by Legate-Maxson** [comment]
arbitrary difficulty for the sake of arbitrary difficulty. If this could effect tools without effecting melee weapons Id support but currently code wise it doesn't support it.

**Comment by cdb-is-not-good** [comment]
> arbitrary difficulty for the sake of arbitrary difficulty. If this could effect tools without effecting melee weapons Id support but currently code wise it doesn't support it.

not really true, no. One would just have to manually go through and make tools other than melee have less health. Though the breaking should be fine for all tools. One should be mindful of their kit before going into a fight, especially since fixing tools is like....really easy and requires very common items.

**Comment by cdb-is-not-good** [comment]
also possum vision seal of approval etc

**Pull request merged**